test/unit/src/common/actions/t_register-name.js

```diff
@@ -198,10 +201,9 @@ describe('register name actions', () => {
 
         beforeEach(() => {
           // XXX check Authorization header
-          scope
-            .post('/api/store/register-name', { snap_name: 'test-snap' })
+          storeScope = nock(conf.get('STORE_API_URL'))
```

Comment:
Do you think it would help to consistently name the SCA api nocks, previously you used storeApi?